### DISCUSSION TITLE
How to disable re-ordering for just one column

### DISCUSSION THREAD

**Discussion by ghost** [serenity-is/Serenity#5784] [Help] [answered]
Hey,
I want column X to always be fixed/frozen at the first position in grid. Other columns can be therefore dragged onto any position except first.

Setting 'pointer-events' to none works great for the column X itself, however, other columns can still be dragged onto it.

Any ideas? Thank you.

**Comment by minhhungit** [comment, marked as answer]
To disable sort feature on a column, you can use attribute [Sortable(false)], for example:

In your columns.cs
```csharp
[Sortable(false)]
public String Title { get; set; }
```

Another way grid.ts:

```javascript
protected getColumns() {
    var columns = super.getColumns();

    let customerCol = Q.tryFirst(columns, x => x.field == "Customer");
    if (customerCol) {
        customerCol.sortable = false;
    }

    return columns;
}
```

To fixed/frozen a column in grid, reference to this demo http://jlynch7.github.io/SlickGrid/examples/example-frozen-columns
